synapse/event_auth.py

```diff
@@ -80,6 +80,31 @@ def validate_event_for_room_version(event: "EventBase") -> None:
     Raises:
         SynapseError if there is a problem with the event
     """
+    # Reject events with stringy power levels if required by room version
+    if (
+        event.type == EventTypes.PowerLevels
+        and event.room_version.msc3667_int_only_power_levels
+    ):
+        for key, value in event.content.items():
+            if key in [
+                "users_default",
+                "events_default",
+                "state_default",
+                "ban",
+                "reject",
```

Comment:
Was this meant to be `redact`? I can't find `reject` in the spec
```suggestion
                "redact",
```